requirements/base.txt

```diff
@@ -2,9 +2,10 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile requirements/base.in
+#    pip-compile '.\requirements\base.in'
 #
-attrs==20.3.0             # via -r requirements/base.in
-iso3166==1.0.1            # via -r requirements/base.in
-jsons==1.3.0              # via -r requirements/base.in
-typish==1.9.1             # via jsons
+attrs==20.3.0             # via -r .\requirements\base.in
```

Comment:
`attrs` helps us to build nice classes with less boilercode. It gives us nice string representations and the option to compare objects of the same class with each other. So far it would be the same as a dataclass.
The main difference I saw so far is, that the inheritance of the classes with optional and required attributes is easier to achieve then with dataclasses.
You can find more [differences between dataclass and attrs on their documentation](https://www.attrs.org/en/stable/why.html#data-classes)